app/controllers/gobierto_admin/gobierto_attachments/api/attachments_controller.rb

```diff
@@ -0,0 +1,97 @@
+module GobiertoAdmin
+  module GobiertoAttachments
+    module Api
+      class AttachmentsController < ::GobiertoAdmin::Api::BaseController
+
+        before_action :find_attachable, only: [:index]
+        before_action :find_attachment, only: [:update, :destroy]
+
+        def index
+          if params[:search_string]
```

Comment:
`attachments = if ... elsif .. else`, to avoid the assignment of the variable three times